pyanaconda/modules/storage/bootloader/efi.py

```diff
@@ -194,14 +195,80 @@ def write_config(self):
 
         super().write_config()
 
+class EFISYSTEMD(EFIBase, SYSTEMD):
+    """EFI Systemd-boot"""
+    _packages_common = [ "efibootmgr", "systemd-udev", "stubby"]
```

Comment:
the stubby name here is going to have to change, I was only looking at the server rpms rather than everything, and there is already a stubby package. I think the plan is to open a package request as sdubby.